spec/schemas/cat.nodes.yaml

```diff
@@ -101,11 +101,10 @@ components:
           type: string
         master:
           description: |-
+            (Deprecated in 2.0.0. To promote inclusive language, use 'cluster_manager' instead.)
             Indicates whether the node is the elected master node.
             Returned values include `*`(elected master) and `-`(not elected master).
           type: string
-          x-version-deprecated: 2.0.0
-          x-deprecation-message: To promote inclusive language, use 'cluster_manager' instead.
```

Comment:
What's the reason for this change? It seems like a backwards step and degradation for our codegen use case (ie. this should eventually get annotated as `@Deprecated` in Java & `[Obsolete]` in C#)

Reply:
This is the only case where these extensions are used in a schema (They only appear in parameters otherwise). So, I thought it was a misuse. Can add these to the validator's vocabulary. Are there other extensions do you think should be added to the vocabulary as well?

Reply:
Added [these](https://github.com/opensearch-project/opensearch-api-specification/pull/294/files#diff-e154da95e179db1faa593dbfd567bccc4f91cf6620672188f6e62c564c88f716R12-R17)